### ISSUE TITLE
[client/rest] Can we add the database size in rest?

### ISSUE THREAD

**Issue by AnthonyLaw** [symbol/symbol#4]
## Current Behaviour

We are unable to tell what is the database size in the current network. It will help node owners to plan for the storage size before they run a node, and information is also helpful for analytics and research, you want to map total state growth over time and cross-reference that with product releases.

## Expected Behaviour

We can add that information to this endpoint `/node/storage`.

**Comment by Jaguar0625** [comment]
Fixed in 67eb6dad266f3c686d59a2747dbf812c2d04ef28.

**Comment by cryptoBeliever** [comment]
rest 2.4.3 `/node/storage` now returns also database statistics

```
{
   "numBlocks":20521,
   "numTransactions":61476,
   "numAccounts":1934,
   "database":{
      "numIndexes":73,
      "numObjects":106734,
      "dataSize":115363089,
      "indexSize":8601600,
      "storageSize":82223104
   }
}
```
